wiki/Skinning/Compendium/en.md

```diff
@@ -3,34 +3,43 @@
 [o!c]: /wiki/shared/mode/catch.png "osu!catch"
 [o!m]: /wiki/shared/mode/mania.png "osu!mania"
 
-# Compendium
-Compendium of all completed osu! skins.
+# Skins Compendium
+This is a compendium of all completed osu! skins
 
-- ![][o!s] means the skin contains osu!standard elements.
-- ![][o!c] means the skin contains osu!catch elements.
-- ![][o!t] means the skin contains osu!taiko elements.
-- ![][o!m] means the skin contains osu!mania elements.
+- ![][o!s] This means the skin contains osu!standard elements
+- ![][o!c] This means the skin contains osu!catch elements
```

Comment:
`- ![][o!c] - skin contains osu!catch elements`

same for ones below

Reply:
Oh, I see. The phrasing and the heading changed for some reason. Looks like the programm for the compendium needs a bit of adjustment. I guess the old text is fine?

Reply:
old text is fine for those, might actually add a hyphen if you want

Reply:
hyphen between `]` and `means`?

Reply:
yeah